package.json

```diff
@@ -82,7 +82,9 @@
     "codecov": "^3.0.0",
     "coffeescript": "^2.0.3",
     "cross-env": "^5.1.1",
+    "elm": "^0.19.0-bugfix2",
```

Comment:
Perhaps change this to `^0.19.0` not sure why they're appending `-bugfix` instead of just bumping the patch version